web/src/views/PublicLanding.tsx

```diff
@@ -39,9 +130,83 @@ export const PublicLanding = () => {
   }
 
   return (
-    <div>
-      <h1>Welcome</h1>
-      <h4>Please log in to proceed</h4>
-    </div>
+    <>
+      <StyledHeader>
+        Welcome
+        <HeaderDescription>
+          to BC Gov's Platforom as a Service( PaaS) Porject Registry
+        </HeaderDescription>
+      </StyledHeader>
+      <HomePageSectionContainer>
+        <StyledSubHeader>Make changes to an existing project </StyledSubHeader>
+        <StyledParagraph>
+          For existing application hosted on Openshift 4 Platform. You can update/change all project
+          details and request project resource quota increases and downgrades (including
+          CPU/RAM/Storage.)
+        </StyledParagraph>
+        <StyledButton onClick={() => keycloak.login({ idpHint: 'idir' })}>Log In</StyledButton>
+      </HomePageSectionContainer>
+      <HomePageSectionContainer>
+        <StyledSubHeader>Register a new project</StyledSubHeader>
+        <StyledParagraph>
+          Use this website if you are a Product Owner for a new cloud-native application and are
+          interested in hosting the app on the Openshift 4 Platform. You can learn about the BCGov's
+          PaaS/Openshift 4 Platform Service here.
+        </StyledParagraph>
+      </HomePageSectionContainer>
+      <HomePageSectionContainer>
+        <StyledSmallHeader>Before you start</StyledSmallHeader>
+        <StyledParagraph>
+          This website is for teams who've attended an onboarding session with the platform team (if
+          you currently host an application on Openshift, you’ve done this already.) If you haven’t
+          attended an onboarding session, please contact the Platform Director
+          (olena.mitovska@gov.bc.ca) to book an onboarding session.
+        </StyledParagraph>
+
+        <StyledacknowledgeMessage active={showWarrningMessage}>
+          <StyledCheckbox
+            name="have attened onboarding session"
```

Comment:
This could do with being more concise, (there is also a typo: attended). Something along the lines of:

```suggestion
            name="attendedOnboardingSession"
```

Reply:
Updated